src/components/learner-credit-management/data/hooks/useSuccessfulCancellationToastContextValue.js

```diff
@@ -0,0 +1,41 @@
+import { useCallback, useMemo, useState } from 'react';
+
+const generateSuccessCancelMessage = (assignmentUuidCount) => {
+  if (assignmentUuidCount > 1) {
+    return `Assignments canceled (${assignmentUuidCount})`;
+  }
+
+  return 'Assignment canceled';
+};
+
+const useSuccessfulCancellationToastContextValue = () => {
+  const [isToastOpen, setIsToastOpen] = useState(false);
+  const [assignmentUuidCount, setAssignmentUuidCount] = useState();
+
+  const handleDisplayToast = useCallback((assignmentUuids) => {
+    setIsToastOpen(true);
+    setAssignmentUuidCount(assignmentUuids);
+  }, []);
+
+  const handleCloseToast = useCallback(() => {
+    setIsToastOpen(false);
```

Comment:
Might be worth ensuring this is covered in a test case(s), to verify there's no future regressions that erroneously keep the `Toast` notification open all the time.